docs/docs/how-tos/login-thru-keycloak-howto.md

```diff
@@ -0,0 +1,76 @@
+---
+id: login-keycloak
+title: Authenticate using Keycloak and launch your server
+description: |
+  A step-by-step guide to logging into your Nebari instance and launching a JupyterLab pod.
+---
+
+# How to authenticate using Keycloak and launch your server
+
+This guide provides a basic overview of how to login to your Nebari instance and launch a JupyterLab pod from the perspective of regular user.
+
+## 1. Connect to your Nebari cluster
+
+Navigate to the cluster homepage, for instance [`https://demo.nebari.dev`](https://demo.nebari.dev).
+
+Once on the site, you will be prompted to login, similar to
+the login page shown in the image below.
+
+![Nebari login screen](/img/how-tos/nebari_login_screen.png)
+
+Nebari uses [Keycloak](https://www.keycloak.org/), an open-source identity and access management tool. Keycloak is a centralized location for administrators to add new users, create user groups and update roles. For more information, see the [How to Configure Keycloak](/how-tos/configure-keycloak-howto.md) docs.
+
+:::warning
+As Nebari **does not** create a "regular" user for you during deployment, only the `root` Keycloak admin user. The following steps assume that your Nebari administrator has already created a Keycloak user in the Keycloak admin console () for you. If not, follow the steps in [Adding new users to Keycloak](/how-tos/configuring-keycloak#adding-a-nebari-user) to create a Keycloak user.
+:::
+
+## 2. Authenticate with your provider
+
+At this stage, it is a little challenging to provide thorough instructions the particulars will depend on your [identity provider](https://www.keycloak.org/docs/latest/server_admin/#_identity_broker) / authentication provider ([LDAP](https://pt.wikipedia.org/wiki/LDAP), [OAuth 2.0](https://oauth.net/2/), passwordless authentication, password-based authentication and many others).
```

Comment:
```suggestion
At this stage, it is a little challenging to provide thorough instructions because the particulars will depend on your [identity provider](https://www.keycloak.org/docs/latest/server_admin/#_identity_broker) / authentication provider ([LDAP](https://pt.wikipedia.org/wiki/LDAP), [OAuth 2.0](https://oauth.net/2/), passwordless authentication, password-based authentication and many others).
```